starter-code/css/main.css

```diff
@@ -6,3 +6,150 @@ Green: #00B172
 White: #FFF
 
 */
+
+body{
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 1.3em;
+  font-weight: lighter;
+  background-color: gray;
+  margin: 0;
+  padding: 0;
+}
+
+.page-container {
+  margin: -45px auto 15px;
+  padding: 0;
+  width: 1200px;
+  background-color: white;
+}
+
+nav {
+  position: fixed;
+  height: 100px;
+  width: 1200px;
+  margin: 0;
+  background-color: white;
+}
+
+nav img{
+  float: left;
+  height: 80px;
+  margin: 10px 50px;
+}
+
+.nav-bar-links {
+  float: right;
+  width: fit-content;
+  height: 80px;
+  margin: 0;
+  padding: 0 50px;
+}
+
+.nav-bar-links li {
+  float: left;
+  height: 45px;
+  padding-top: 30px;
+  margin: 10px 15px;
+  list-style-type: none;
+  font-size: 1.1em;
+
+}
+
+header {
+  margin: 0px;
+  height: 800px;
+  background-image: url(./../images/landing.jpg);
+  background-size: contain;
+  color: #fff;
+  font-size: 1.5em;
+}
+
+header h1 {
+  text-align: center;
+  position: relative;
+  top: 300px;
+}
+
+header p {
+  text-align: center;
+  position: relative;
+  top: 300px;
+}
+
+.whats-on h2 {
+  font-size: 1.8em;
+  text-align: center;
+  margin: 50px 0 100px;
+  text-decoration: underline #00b172;
+}
+
+.col-group {
+  width: 80%;
+  margin: 0 auto;
+  text-align: center;
+}
+
+.col-group > * {
+  margin-bottom: 80px;
```

Comment:
this would have left it spot on 
`
    margin-bottom: 80px;
    float: left;
    margin-left: 50px;
    width: 26%;
`